templates/keyword.html

```diff
@@ -1,15 +1,48 @@
 {{ define "content" }}
 
-{{ include "partials/flashes" }}
+<div class="keyword-container">
+  <h2>Upload CSV</h2>
+  
+  {{ include "partials/flashes" }}
+  
+  <form class="keyword-upload-form space-y-6" action="/keyword" enctype="multipart/form-data" method="POST">
+    <input name="file" type="file" required accept=".csv">
+  
+    <button type="submit">
+      Upload
+    </button>
+  </form>
 
-<h2>Upload CSV</h2>
+  {{ if .data.keywords }}
+    <div class="mt-8 -my-2 overflow-x-auto sm:-mx-6 lg:-mx-8">
+      <div class="py-2 align-middle inline-block min-w-full sm:px-6 lg:px-8">
+        <div class="shadow overflow-hidden border-b border-gray-200 sm:rounded-lg">
```

Comment:
That seems like a lot of nested divs 😵 You have a case of divitis 😷 That issue is often caused by wanting to use all CSS helpers. It's also okay to mix helpers with custom styles:

![image](https://user-images.githubusercontent.com/696529/101457826-9ab70600-3968-11eb-864f-483fbc0e84be.png)

Also prefer creating a custom component to wrap the Tailwin styles:

```
<div class="list-keyword">
  <table class="list-keyword__table"></table>
</div>
```

Reply:
Thank you for clarify this
Fixed in 85fe1a1de436bd5c897893b097b042e8359e22ac